package.json

```diff
@@ -79,6 +80,7 @@
     "color-string": ">=1.9.0",
     "del": "^7.1.0",
     "dot-prop": ">=9.0.0",
+    "esbuild": "^0.21.5",
```

Comment:
I think we're not using some of the babel dependencies anymore after this change. See what all you can remove.